preferences/linux/kustomization.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../base
+
+components:
+  - ./metadata
+  - ../components/diskbus-virtio-blk
+  - ../components/interfacemodel-virtio-net
+  - ../components/rng
```

Comment:
If Linux generic is wirth virtio
Then we want a second "Generic" which is scsi + e1000 or rtl8139.

cc @dominikholler

Reply:
> If Linux generic is wirth virtio Then we want a second "Generic" which is scsi + e1000 or rtl8139.

I was about to say that this would just use the VMI defaults but I see we use a default interface model of virtio.

I wouldn't be against introducing a legacy-linux generic preference using SCSI and e1000.

Reply:
Great! I will update the PR.

Reply:
we really must have a really generic one without virtio.

Reply:
Maybe we can onboard another generic, i.e., linux (no virtio), linux-viritio and linux-legacy. WDYT?
Open question, should we keep the interfacemodel virtio in the linux preference?

Reply:
I tend to agree to Lee. I've included the `linux-legacy` preference.

Reply:
@fabiand What is more suited as disk bus for "legacy generic"? `sata` or `scsi`?